web/views/FcLayoutDrawerMap.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <main class="fc-layout-drawer-map flex-fill flex-container-column">
+    <section class="panes flex-fill flex-container-row">
+      <router-view
+        v-show="drawerOpen"
+        class="pane-display"></router-view>
```

Comment:
can we call this `pane-drawer` everywhere?  (see also: `.pane-display-toggle` in `PaneMap`.)

also we can get rid of `LayoutMain` and `LayoutViewData`, which are both obsolete and unused at this point.